src/apps/beefy/positions.test.ts

```diff
@@ -1,5 +1,100 @@
-import { BaseBeefyVault } from './api'
-import { getDailyYieldRatePercentage } from './positions'
+import { TFunction } from 'i18next'
+import {
+  BaseBeefyVault,
+  getApyBreakdown,
+  getBeefyPrices,
+  getBeefyVaults,
+  getTvls,
+} from './api'
+import hook, { getDailyYieldRatePercentage } from './positions'
+import { NetworkId } from '../../types/networkId'
+import { Address } from 'viem'
+
+const mockT = ((x: string) => x) as TFunction
+
+const mockReadContract = jest.fn()
+jest.mock('../../runtime/client', () => ({
+  getClient: jest.fn(() => ({
+    readContract: mockReadContract,
+  })),
+}))
+
+jest.mock('./api.ts')
+
+const mockBeefyVault1: BaseBeefyVault = {
+  id: 'vault1',
+  name: 'Vault 1',
+  type: 'auto',
+  subType: 'concentrated',
+  token: 'vault1Token',
+  tokenAddress: '0x123456789' as Address,
+  tokenDecimals: 8,
+  tokenProviderId: '',
+  earnedToken: '',
+  earnContractAddress: '' as Address,
+  status: '',
+  platformId: '',
+  assets: [],
+  risks: [],
```

Comment:
Explicit unit testing for safety score already exists so left that out of implementation here, but can add in if wanted.